### ISSUE TITLE
Shift + Cursor Keys should also mark the last selected file

### ISSUE THREAD

**Issue by sagamusix** [doublecmd/doublecmd#281]
I understand that the current behaviour is probably due to how TotalCommander does things, and maybe there's even an option to turn it off that I haven't found yet, but the following behaviour annoys me somewhat especially since it isn't present in other programs also following the orthodox file manager pattern (e.g. WinSCP): When I select a file and then extend the selection using Shift+Up/Down, the last selected file is not part of the selection. However, if I draw exactly the same selection by Shift-Clicking on the last file, then it is part of the selection. I'd greatly appreciate this also was the case when using Shift+Cursor Keys.

**Comment by WhiteChairFromIkea** [comment]
I agree, there must be an option for that. Now, when I have learned how it works there is no problem anymore, but this behavior is a black sheep among others, need to refink my workflow in regular explorers and different computers..

Also, this was one of the first issues I have reported in unofficial repo, when started using dc 😁

**Comment by goyalyashpal** [comment]
This issue can be closed as a duplicate of above mentioned double-commander/doublecmd#13

**Comment by goyalyashpal** [comment]
oh - uh oh, i _[AGAIN]_ missed the details about repos - what's happening to me. goshhhhh. sorry.

double-commander vs doublecmd

[again]: https://github.com/doublecmd/doublecmd.github.io/issues/52#issuecomment-1427072782

**Comment by goyalyashpal** [comment]
_moving over my input VERBATIM from over there_

----
> Now, when I have learned how it works there is no problem anymore ...
\- @ WhiteChairFromIkea on 2021.12.xx at https://github.com/doublecmd/doublecmd/issues/281

the correct way to select files in doublecmd is to use `space bar` and arrow keys. it works wonders once u leave behind the windows explorer's legacy - which u already have 😉 

Some notes: 
- instead of the normal arrow keys, u can also use the numpad-navigation keys (i.e. when numlock off) to perform same task.
- so:
- `shift + down arrow` work the same as pressing `Insert` or `shift + num2 (numlock off)`
- `shift + up arrow` work same as pressing `shift + num8 (numlock off)`

This is slightly better than the `space` + `arrow` as it brings down the keypress count from 2 every time to just 1 with modifier pressed throughout. However, i have one suggestion which _i think_ will improve ur use case further.

**Comment by sagamusix** [comment]
Thanks but no thanks. I have already outlined in my initial post why I disagree with Double Commander's behaviour and why I think there should be an option to toggle between WinSCP/Explorer/etc.-like behaviour and TotalCommander-like behaviour.

**Comment by goyalyashpal** [comment]
Just to be clear, so, u are asking for _only the proposed solution_ i.e. the underlined problem is not the main concern, right?

**Comment by WhiteChairFromIkea** [comment]
> Just to be clear, so, u are asking for _only the proposed solution_ i.e. the underlined problem is not the main concern for u, right?

The point is that very odd (for win explorer user) out of the box experience is frightening newcomers. The option to switch between explorer like and orthodox like behaviour would be very nice, this is a valid feature request. Since this is an open source project, maybe someone will implement it. Host of this project is really friendly and accepts pull requests. Btw, thanks for tips @goyalyashpal, it is always welcome when trying to help 🙂

**Comment by goyalyashpal** [comment]
yes, that's why i asked if this is a feature request or aimed at problem.

**Comment by sagamusix** [comment]
As said, I would appreciate if there was *an option* to toggle this behaviour - hence this is a feature request.

**Comment by goyalyashpal** [comment]
ohkay, good.
